packages/coordinator/tests/services/verification/signature-verification.spec.ts

```diff
@@ -43,6 +47,33 @@ describe("verifySignatureHasCorrectFormat...", () => {
   })
 })
 
+describe("verify if certificate is not revoked...", () => {
```

Comment:
It looks like the pattern used in this spec folder is describe block is function name being tested 
test block is each behaviour expected. We should follow that pattern